cmd/antrea-controller/controller.go

```diff
@@ -142,6 +145,12 @@ func createAPIServerConfig(kubeconfig string,
 		return nil, err
 	}
 
+	if err := os.MkdirAll(path.Dir(apiserver.TokenPath), os.ModeDir); err != nil {
+		return nil, fmt.Errorf("error when creating dirs of token file")
```

Comment:
suggest "error when creating directory for apiserver token file"

Reply:
you are not adding the actual error to the message?

Reply:
My bad, updated.